package.json

```diff
@@ -31,7 +31,9 @@
   },
   "homepage": "https://github.com/moscajs/aedes-otel-instrumentation#readme",
   "scripts": {
-    "test": "node --require @esbuild-kit/cjs-loader --test-reporter=spec --test test/*.spec.ts",
+    "otel-test": "node --require @esbuild-kit/cjs-loader --test-reporter=spec --test test/*.spec.ts",
+    "aedes-test": "npm run build && node --require ./test/tracing --test test-aedes/*.js",
+    "test": "npm run otel-test && npm run aedes-test",
```

Comment:
now the tests from aedes are running with tracing config enabled (it also give an idea of the performance impact of enabling tracing).

Reply:
That's awesome 👍🏼